Detectors/TPC/workflow/src/RecoWorkflow.cxx

```diff
@@ -218,20 +222,6 @@ framework::WorkflowSpec getWorkflow(CompletionPolicyData* policyData, std::vecto
                                                    std::vector<int>(1, 0),
                                                    &hook},
                                                  false));
-  } else if (inputType == InputType::EncodedClusters) {
-    // TODO: need to check if we want to store the MC labels alongside with encoded clusters
-    // for the moment reading of labels is disabled (last parameter is false)
-    specs.emplace_back(o2::tpc::getPublisherSpec(PublisherConf{
-                                                   "tpc-encoded-cluster-reader",
-                                                   "tpcrec",
-                                                   {"clusterbranch", "TPCEncodedClusters", "Branch with TPC encoded clusters"},
-                                                   {"clustermcbranch", "TPCClusterNativeMCTruth", "MC label branch"},
-                                                   OutputSpec{"TPC", "ENCCLUSTERS"},
-                                                   OutputSpec{"TPC", "CLNATIVEMCLBL"}, // This does not work with labels!
-                                                   std::vector<int>(1, 0),
-                                                   std::vector<int>(1, 0),
-                                                   &hook},
-                                                 false));
```

Comment:
If you remove this, could you also remove the InputType `EncodedClusters` in the header, and the input mapping "encoded-clusters" --> ... in the beginning of RecoWorkflow.cxx

Reply:
ok